deploy/v2/terraform/modules/hdb_node/main.tf

```
@@ -167,55 +167,70 @@ resource "azurerm_managed_disk" "data-disk" {
 }
 
 # Manages Linux Virtual Machine for HANA DB servers
-resource "azurerm_linux_virtual_machine" "vm-dbnode" {
-  for_each            = local.dbnodes
-  name                = each.value.name
-  computer_name       = each.value.name
-  location            = var.resource-group[0].location
-  resource_group_name = var.resource-group[0].name
-  availability_set_id = azurerm_availability_set.hana-as[0].id
-  network_interface_ids = [
-    azurerm_network_interface.nics-dbnodes-admin[each.key].id,
-    azurerm_network_interface.nics-dbnodes-db[each.key].id
-  ]
-  size                            = lookup(local.sizes, each.value.size).compute.vm_size
-  admin_username                  = each.value.authentication.username
-  admin_password                  = lookup(each.value.authentication, "password", null)
-  disable_password_authentication = each.value.authentication.type != "password" ? true : false
-
-  dynamic "os_disk" {
+# VIRTUAL MACHINES ================================================================================================
+
+# TODO:
+# Currently there is an issue with SSH logon to HANA database VM if it is created with azurerm_linux_virtual_machine
+# Will migrate this code to use azurerm_linux_virtual_machine when the above issue fixed
+
+# Creates HANA database VM using azurerm_virtual_machine
+resource "azurerm_virtual_machine" "vm-dbnode" {
+  for_each                      = local.dbnodes
+  name                          = each.value.name
+  location                      = var.resource-group[0].location
+  resource_group_name           = var.resource-group[0].name
```

Availability set association is missing in this revert.

```suggestion
  resource_group_name           = var.resource-group[0].name
  availability_set_id           = azurerm_availability_set.hana-as[0].id
```

---

LGTM 👍

---

> Availability set association is missing in this revert.

Sorry, forgot it again. Now added. Thanks!